### DISCUSSION TITLE
History Graph: Change negative Values to Positive

### DISCUSSION THREAD

**Discussion by N1kless** [Zefau/ioBroker.jarvis#1667] [Q&A]
Hey everyone,

I am just trying to put my PV System Data  (Power Generation, Household Load, Grid Power usage/sale) into one History Graph. Some values like Load are negative and I would like them to be positive. Is there a way to multiply with minus one? Flot has this function...

Beste regards,
Nik

**Comment by mcuiobroker** [comment]
Not yet. This Function needs a callback for values. This is one of an open Features.
https://github.com/Zefau/ioBroker.jarvis/issues/1459

**Comment by Zefau** [comment]
Now available via v3.1
